### ISSUE TITLE
Unable to click on UI elements in Extended Mode

### ISSUE THREAD

**Issue by Pywinautousr** [pywinauto/pywinauto#722]
I am Launching an app (any UWP app) in primary monitor and moving to Extended monitor. If I try click by click_input() any element on the app it is clicking on right bottom of  the extended monitor. This same issue seen even when I directly launch the application on extended monitor. However if I print controls by print_control_identifiers(), printing controls on the app.

The weird behavior I see is, If already open an application get the control (by App.connect or Desktop) I am able to click and do my job. With the same results I tried to launch another script from major script, it is controlling elements on the app UI.

If I launch an app by one program and try to control from my major script, again it giving trouble  to control UI elements.

My sample script looks like :

class ControlUI:
     def _init_(self):
         dlg_desktop= None
    def LaunchApp(self, disp_mode):
         if disp_mode == "Extend":
               self.MoveWindow()
         self.dlg_desktop = Desktop(backend="uia")["MyApp"]
         self.dlg_desktop.child_window(title="Sign In", control_type="Button").click_input()

**Comment by airelil** [comment]
Sounds as a bug. As a wild guess it might be related to DPI scaling, or we may getting negative values of GUI coordinates and failing on it (In that case you may try swapping the places of your primary and secondary display). Can you enable a bit more logging for your script and send us the output? To enable the detailed prints, add to the top of your file:
```python
from pywinauto import actionlogger
actionlogger.enable()
```

**Comment by Pywinautousr** [comment]
@airelil Yes, this is due to dpi scaling issues and co-ordinates were gone bad. The moment I change extended scaling to primary monitor it started working.

So, will wait for 0.7.0 version for the fix.

Thanks for addressing the issue.

**Comment by airelil** [comment]
Glad you figured it out. Mind sharing a couple of logs with abovementoned actionlogger enabled? Could be interesting to see a good and a bad example.

**Comment by Pywinautousr** [comment]
Hi @airelil , 
Unfortunately at this moment i may not share the logs as it is proprietary application which is not in market and I just got to know about pywinauto, so I am just trying test my app with the same for my reference. 

Thank you guys for your contributions, I found good advantages of pywinauto and will post logs soon after our product is in market :)
I have one more query on win32 windows will raise another ticket for that.
